### PR TITLE
addon-knobs radio input

### PR DESCRIPTION
**Pull request by astrotim**
Issue:

## What I did

Added 'radio' knob to addon-knobs. This is functionally the same as the 'select' knob, however the UI is build with `<input type="radio">` so the options are presented in a horizontal list with radio inputs. 

An ideal use-case for this feature is a theme switcher - e.g. light and dark theme, etc. This is a superior user experience than the 'select' knob, as the selection can be more easily changed by single mouse clicks or using the arrow keys. 

## How to test

Is this testable with Jest or Chromatic screenshots?

I have included the [Radio.js test file](https://github.com/astrotim/storybook/blob/addon-knobs-radio-input/addons/knobs/src/components/__tests__/Radio.js) for testing with Jest. 

Does this need a new example in the kitchen sink apps?

No.

Does this need an update to the documentation?

Yes. The README has been updated with relevant docs.

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`

**Example**

example

### CODE REVIEW

**Review by ndelangen — CHANGES_REQUESTED** [review]
Can you check the comment I left about the value?

Also I think we could use a nice example of this new knob in the examples/official-storybook story for knobs!

**Comment by ndelangen** [comment]
@astrotim I like it, good addition to the knobs addon for sure!

**Comment by igor-dv** [comment]
But we already have an open PR for this : #3894

**Comment by ndelangen** [comment]
I see, great minds think alike!

Perhaps you'd want to seize the opportunity to work together on this feature?
@brycemhammond @astrotim what can I do to assist you?

**Comment by astrotim** [comment]
Thanks @ndelangen, I have looked at PR #3894, what a coincidence! A key difference I can see straight up is the different name for the knob itself. 

Do you guys want to make a call whether you prefer ‘radio’ or ‘radioButton’?

**Comment by ndelangen** [comment]
I prefer `radios` or `radio`

File: addons/knobs/README.md [hunk 1]
```diff
@@ -264,6 +264,31 @@ const value = select(label, options, defaultValue, groupId);
...
+  Rainbow: ['red', 'orange', 'etc'],
+  None: null,
+};
+const defaultValue = 'red';
+const groupId = 'GROUP-ID1';
+
+const value = radio(label, options, defaultValue, groupId);
+```
+
+> You can also provide options as an array like this: `['red', 'blue', 'yellow']`
```
[the remark this answers is not in the document]
Comment (Hypnosphi) [reply addons/knobs/README.md]: Again, this matches what we have in select, and we don't have much trouble there

**Comment by igor-dv** [comment]
TBH, I think we should finish the PR @brycemhammond started since it was the first one. 
Also. @nfiniteset and @brycemhammond created an [issue](https://github.com/storybooks/storybook/issues/3872) even earlier.

**Comment by ndelangen** [comment]
Hey @astrotim I hope you don't mind:
I just merged #3894 so this feature is in.

Thank you for your effort, truly appreciated!

**Comment by astrotim** [comment]
No problem at, I'm happy to see it added either way.

Congrats on becoming a member of the Chromatic team, by the way 👍🏼

**Pull request closed without merging**